cmd/kubeadm/app/cmd/phases/init/kubelet.go

```diff
@@ -68,6 +69,29 @@ func runKubeletStart(c workflow.RunData) error {
 		kubeletphase.TryStopKubelet()
 	}
 
+	// Check if the user has added the kubelet --root-dir flag.
+	// If yes, symlink default kubelet directory to --root-dir.
```

Comment:
why not just let the user do this manually instead of kubeadm doing it?

Reply:
Maybe potentially minimize operation and maintenance efforts and could reduce the risk  I thought.